tsconfig.json

```diff
@@ -25,9 +25,5 @@
 		// "noUncheckedIndexedAccess": true, // TODO: Enable.
 		"noPropertyAccessFromIndexSignature": true,
 		"useDefineForClassFields": true,
-	},
-	"exclude": [
-		// Ignore `WriteableDeep` error temporarily, remove when #833 are fixed.
-		"test-d/writable-deep.ts",
```

Comment:
Would you be able to add a test to `test-d/writable-deep.ts` too? So it doesn't regress again in the future.

Reply:
@sindresorhus Sure, there is a test already, but most tests in both of these are indirect ones which gives some cascading one, I'll add en explicit one 👍

Reply:
@sindresorhus Done ✔️